docs/repo/property-pages/visibility-conditions.md

```diff
@@ -10,8 +10,9 @@ Visibility conditions are not interpreted by the back-end, which treats them as
 
 ## Specification
 
-There are three kinds of visibility conditions:
+There are three four of visibility conditions:
```

Comment:
Err, "four kinds" not "three four".

Reply:
Also singular _condition_.

Reply:
as seen by the sentences I wrote, I can english very well @tmeschter 😄 
@drewnoakes plural conditions is correct in american english

Reply:
Thanks. TIL there is a difference of opinion on this! https://english.stackexchange.com/questions/5539/types-of-things-vs-types-of-thing

I'd write:
- Four visibility conditions
- Four kinds of visibility condition

Apparently Edge would write:

![image](https://user-images.githubusercontent.com/350947/182272590-1c84af8e-4339-44c1-9d5e-58ed48b3caf6.png)

😆